README.md

```diff
@@ -1,5 +1,5 @@
 # PaperLib
-Plugin Library for interfacing with Paper Specific API's with graceful fallback that maintains Spigot Compatibility, such as Async Chunk Loading.
+PaperLib is a plugin library for interfacing with Paper specific APIs (such as async chunk loading), with graceful fallbacks maintaining compatibility with Bukkit/Spigot.
```

Comment:
I would remove Bukkit here

Reply:
Well, I suppose Spigot is more well known these days. I can remove it if you want, I just figured that it'd be nice to explicitly state it doesn't make your plugin require Spigot's API either, since it still works on CB servers.